src/cli/mod.rs

```diff
@@ -86,6 +91,8 @@ pub enum Command {
     Remove(remove::Args),
     #[clap(visible_alias = "i")]
     Install(install::Args),
+    #[clap(visible_alias = "u")]
```

Comment:
```suggestion
```
I would vote to not alias it already. If we add `upgrade` it might not be helpful as these will probably be used in scripts. 
I also checked `cargo` `poetry` and `rye` and they also don't alias it.